ehrapy/plot/_sa_plot.py

```diff
@@ -0,0 +1,229 @@
+from __future__ import annotations
+
+from xmlrpc.client import Boolean
+
+import matplotlib.pyplot as plt
+import numpy as np
+from anndata import AnnData
+from lifelines import KaplanMeierFitter
+from matplotlib.axes import Axes
+from numpy import ndarray
+from statsmodels.regression.linear_model import RegressionResults
+
+import ehrapy as ep
+
+
+def ols(
+    adata: AnnData | None = None,
```

Comment:
Because this function can also be used for only plotting lines defined by slope and intercept. For example, in the notebook, one plot only shows the ols results and no scatter plot. Also, I added several examples to this function, and the third example is exactly this case.
![Screenshot 2022-05-12 at 22 13 11](https://user-images.githubusercontent.com/88339105/168160208-401e2547-99ef-4e98-8f06-b665fd635876.png)

Reply:
Great! This sounds fair :)
Could also throw an error if none of the options to plot something is passed?